mars/deploy/oscar/tests/test_local.py

```diff
@@ -597,6 +597,36 @@ def f1(interval: float, count: int):
     assert info.progress() == 1
 
 
+@pytest.mark.asyncio
+async def test_session_get_progress(create_cluster):
+    session = get_default_async_session()
+    assert session.address is not None
+    assert session.session_id is not None
+
+    raw = np.random.rand(100, 4)
+    t = mt.tensor(raw, chunk_size=5)
+
+    def f1(c):
+        time.sleep(2)
+        return c
+
+    t1 = t.sum()
+    r = t1.map_chunk(f1)
+    info = await session.execute(r)
```

Comment:
How long does this test case take? I noticed that there is a `time.sleep(2)` in the `f1`.

Reply:
It takes about 5s on mars executor, 10s on ray executor.

Reply:
Can you make it faster? Ray backend is slower because:

- import ray
- start ray cluster
- ray kill idle worker (we can config `kill_idle_workers_interval_ms ` to 0, please refer to https://github.com/ray-project/ray/blob/master/src/ray/common/ray_config_def.h#L399)

Reply:
Thanks for you reminder. I'll fix it.